knowledge_graph/routes.py

```diff
@@ -72,9 +59,15 @@ def query() -> Tuple[Response, int]:
 
 
 @app.route("/questions", methods=["GET"])
+@auto.doc()
 def get_questions() -> Tuple[Response, int]:
+    """
+    Returns the list of available questions that can be presented to
+    the user.
+    """
     try:
-        with open("schwartz_questions.json") as json_file:
+        file = os.path.join(os.getcwd(), "json_files/schwartz_questions.json")
```

Comment:
this needs fixing too for good style. 
Should be:
`file = os.path.join(os.getcwd(), "json_files","schwartz_questions.json")`